flink-connectors/flink-connector-hbase/src/main/java/org/apache/flink/connector/hbase/source/HBaseInputFormat.java

```diff
@@ -36,6 +35,10 @@
 
 	private static final long serialVersionUID = 1L;
 
+	public HBaseInputFormat(org.apache.hadoop.conf.Configuration hConf) {
```

Comment:
Could you add some Javadoc on this? What is the `hConf` and what values should be contained in the `hConf`?